app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="navbar<%= '_center' if current_page?(groups_path) %>">
+  <%= link_to @arrow_path do%>
+    <%unless current_page?(groups_path)%>
+      <%= image_tag 'arrow.svg', class: 'arrow'%>
+    <%end%>
+  <% end %>
+  <p><%= "#{@path_description}"%></p>
+
```

Comment:
@HFG43 [OPTIONAL] I also suggest increasing the font size here a little more so that the page title will look better and easier to read